release.sh

```diff
@@ -1164,7 +1165,8 @@ set_build_version() {
 	local toc_game_type version
 
 	if [[ -z "$game_version" ]]; then
-		for path in "${!toc_interfaces[@]}"; do
+		readarray -t sorted < <(for a in "${!toc_interfaces[@]}"; do echo "$a"; done | sort -r)
+		for path in "${sorted[@]}"; do
```

Comment:
This is the fix which makes it the extension less .toc doesn't go override the extension ones by sorting/having a consistent order